airflow/jobs/scheduler_job_runner.py

```diff
@@ -393,7 +394,8 @@ def _executable_task_instances_to_queued(self, max_tis: int, session: Session) -
                     of=TI,
                     session=session,
                     **skip_locked(session=session),
-                ).all()
+                )
```

Comment:
This makes the variable’s annotation wrong

Reply:
Fixed